app/services/signInRequest.js

```diff
@@ -0,0 +1,12 @@
+import fetch from 'isomorphic-fetch';
+
+const signInRequest = async data => {
+  const res = await fetch('http://localhost:3001/api/signin', {
```

Comment:
Please add a config with the base url. You want to easily change this from one env to another and also don't want to write this string in every file where you need it.